pkg/manifests/manifests.go

```diff
@@ -227,6 +244,42 @@ func newIngressControllerService(conf *config.Config) *corev1.Service {
 	}
 }
 
+func newIngressControllerServicePrivateDNS(conf *config.Config) *corev1.Service {
```

Comment:
extracting several of the magic strings from the Private and Normal version of this function will reduce opportunities for drift

Reply:
good call!